packages/muelu/src/Utils/MueLu_UtilitiesBase_def.hpp

```diff
@@ -623,22 +632,9 @@ UtilitiesBase<Scalar, LocalOrdinal, GlobalOrdinal, Node>::
     GetMatrixOverlappedDiagonal(const Matrix& A) {
   // FIXME_KOKKOS
```

Comment:
This comments is addressed now, no?

Reply:
Ah shoot. Yes, yes it is. I'll remove this tomorrow.